test/app-tap/gh-2717-no-quit-sigint.test.lua

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env tarantool
+
+local fiber = require('fiber')
+local tap = require('tap')
+local popen = require('popen')
+
+--
+-- gh-2717: tarantool console quit on sigint
+--
+
+local TARANTOOL_PATH = arg[-1]
+
+local res = tap.test('gh-2717', function(test)
+    print(TARANTOOL_PATH)
+
+    test:plan(2)
```

Comment:
Both cases are failed in CI (I looked at out_of_source job).

Reply:
Now I see that there is only one workflow that runs tests. It seems that all tests are passed there.

Reply:
Add `full-ci` label to the PR to trigger full testing (it is expected, when the patchset is close to push).

Reply:
I'll resolve this comment, because I don't see fails in CI at the moment.